packages/clerk-js/src/ui/components/SignIn/SignInFactorOnePasswordCard.tsx

```diff
@@ -24,9 +24,9 @@ const usePasswordControl = (props: SignInFactorOnePasswordProps) => {
     label: localizationKeys('formFieldLabel__password'),
   });
 
-  const resetPasswordFactor = signIn.supportedFirstFactors.find(
-    ({ strategy }) => strategy === 'reset_password_code',
-  ) as ResetPasswordCodeFactor | undefined;
+  const resetPasswordFactor = signIn.supportedFirstFactors.find(({ strategy }) =>
+    strategy.includes('reset_password'),
```

Comment:
🔧 As mentioned previously, it would be better to be more specific here and check for `reset_password_email_code` and `reset_password_phone_code`.
Given that we do this check in multiple places, it might worth extracting it to a helper function.